databases/week2/sql_Week2Part1.sql

```diff
@@ -0,0 +1,53 @@
+-- HOMEWORK Part 1: Working with tasks
+
+USE hyf_lesson2;
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task
```

Comment:
You're only supposed to insert certain fields, and `id` for instance is not one of them ;)

In the particular case of the `id`, in general it is auto incremented, so you shouldn't insert it manually, no need to.